packages/shared/src/components/Sidebar.spec.tsx

```diff
@@ -54,6 +69,25 @@ const renderComponent = (onboardingStep = -1): RenderResult => {
   );
 };
 
+it('should remove red dot for filter alert when there is a pre-configured feedSettings', async () => {
```

Comment:
Where do we check for the actual call for the API?

Reply:
I actually tried initially, but for some reason even when the `alerts` data was updated already, the mock doesn't do the rerender for the components making the `useTagsAndSources` use the old data. I can't somehow make it possible with the test case. Though locally, it works as intended, so I mocked whether the function will be called.

Reply:
Ok that's tricky but let's keep it this way for now